src/docs/reference/pricing.md

```diff
@@ -2,37 +2,225 @@
+
+    As a trial user, you can always spin-up Plugins. However, to deploy code, you must have a verified account. [Read about verification](#verification).
+
+**What’s the difference between the Limited Trial and the Full Trial?**
+
+    If you connect your GitHub account, and we are able to verify it against a set of parameters, you will be on the Full Trial where you can deploy both code and plugins.
+
+    If you do not connect a GitHub account, or we are not able to verify your account, you will be on the Limited Trial, where you can only deploy plugins.
+
+    While you’re on the Limited Trial, you can initiate verification at any time in order to access the Full Trial experience.
+
+**How far will the $5 one-time Trial grant last?**
+
+    The longevity of your one-time trial grant depends on how many resources you consume. A project that uses most of the resources available on the Trial plan 24/7 will consume the credits in roughly a month, while a simple webserver may be able to run for several months before running out.
+
+### Usage
+
+**How is usage billed?**
+
+    Railway charges for the core components of compute that you consume. These include:
+
+    - GB RAM used. Billed by the minute.
+    - Cores of vCPU used. Billed by the minute.
+    - GB of Network Egress. Billed by the kb.
+
+    Railway offers additional add-ons that are usage-based:
+
+    - GB Volume. Billed by the minute.
+
+    Prices are listed here.
+
+**What will it cost to run my app?**
+
+    With Railway, you are only billed for the resources you consume at any given time. If you want to approximate the cost of running your app, try our calculator.
+
+**How does included usage work on the Hobby plan?**
+
+    Your Hobby plan subscription includes $5 of usage per month. If your total usage at the end of your billing period is $5 or less, you will not be charged for usage. If your total usage exceeds $5 in any given billing period, you will be charged the delta.
+
+**Does included usage on the Hobby plan accumulate over time?**
+
+    Included usage is reset at the end of every billing cycle and does not accumulate over time.
+
+**Can I set limits for maximum usage within a billing period?**
+
+    Usage limits are coming to Railway. In the meantime, if you are on the Hobby plan and you want a guarantee that you keep under a certain spending threshold, you can set up credits as your payment method.
+
+### Billing
+
+**What is the difference between subscription and usage?**
+
+    There are two main components to your bill:
+
+    - **Subscription**: Cost of the plan you’re on (`cost per seat` * `purchased seats`)
+    - **Usage**: Cost of the resources you’ve consumed: (`cost per unit` * `used units`)
+
+**How do I view and manage my subscription?**
+
+    To view and manage your subscription, visit your account page.
```

Comment:
Should link to /account/billing

Reply:
Perhaps rephrase to "Visit the billing section of the Railway dashboard" for consistency?